docs/pages/converting_spectra/fits_header_modify.ipynb

```diff
@@ -0,0 +1,192 @@
+       "BITPIX  =                    8 / array data type                                \n",
+       "NAXIS   =                    0 / number of array dimensions                     \n",
+       "EXTEND  =                    T                                                  \n",
+       "EXTNAME = 'PRIMARY '           / name of this extension                         \n",
+       "VOCLASS = 'Spectrum-1.0'       / VO Data Model                                  \n",
+       "VOPUB   = 'SIMPLE Archive'     / VO Publisher ID URI                            \n",
+       "TITLE   = 'IRS Spectra of Brown Dwarfs from Suarez & Metchev 2022' / data set ti\n",
+       "OBJECT  = '2MASS J21442847+1446077' / name of observed object                   \n",
+       "RA      =    326.1186316666667 / [deg] Pointing position                        \n",
+       "DEC     =    14.76883277777778 / [deg] Pointing position                        \n",
+       "TMID    =            2454669.5 / [d] MJD mid expsoure                           \n",
+       "TSTART  =            2454680.5 / [d] MJD start time                             \n",
+       "TSTOP   =            2454658.5 / [d] MJD stop time                              \n",
+       "TELAPSE =                42240 / [s] Total elapsed time                         \n",
+       "SPEC_VAL= 'Mid-Infared'        / [angstrom] Characteristic spec coord           \n",
+       "SPEC_BW =    8.951589999999999 / [[um]] Width of spectrum                       \n",
+       "TDMIN1  =              5.21725 / [[um]] starting wavelength                     \n",
+       "TDMAX1  =             14.16884 / [[um]] ending wavelength                       \n",
+       "AUTHOR  = 'Suarez, Genaro; Metchev, Stanimir' / author of the data              \n",
+       "VOREF   = '2022MNRAS.513.5701S' / bibcode                                       \n",
+       "DATE    = '2022-08-25'         / date of file creation                          \n",
+       "INSTRUME= 'IRS     '           / name of instrument                             \n",
+       "DATE-OBS= '2008-08-02'         / date of the observation                        \n",
+       "REFERENC= ' 10.1093/mnras/stac1205' / bibliographic reference                   \n",
+       "TELESCOP= 'Spitzer '           / name of telescope                              \n",
+       "HISTORY 'Original file: 2144%2B1446_IRS_spectrum.dat', This file generated by SI\n",
+       "HISTORY MPLE-db/scripts/ingests/rewrite_spectra.py                              "
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "header_2144 = getheader('../../../../astrodb_utils/tests/data/2MASS+J21442847+1446077.fits')\n",
+    "header_2144"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## See if any keywords are missing"
```

Comment:
a little text here saying that you expect something like an unboundlocalerror if a keyword is missing could be useful. just so that they know the notebook isn't broken when there's an error :),

Reply:
There should not have been an error!  Re-ran the notebook with the fixes from recently merged PR.